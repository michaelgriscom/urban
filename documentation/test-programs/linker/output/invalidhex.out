H:InvalidHex:0000:0010:0100:2012151,16,31,12:URBAN-LLM:0019:InvalidHex
T:000002:F8000000:InvalidHex
T:000003:F8000000:InvalidHex
E:0004:0002:InvalidHex
