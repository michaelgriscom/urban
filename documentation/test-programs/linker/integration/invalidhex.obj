H:InvalidHex:000000:0100:0010:2011300,06,06,07:1000:URBAN-ASM:InvalidHex
L:VAR1:notHex:InvalidHex
T:notHex:00000000:C:1:InvalidHex
M:000000:1:+:VAR1:InvalidHex
T:notHex:00000000:C:1:InvalidHex
M:notHex:1:+:VAR1:InvalidHex
T:000002:notHex8d:C:1:InvalidHex
M:000002:1:+:VAR1:InvalidHex
T:000003:notHex8d:C:1:InvalidHex
M:notHex:1:+:VAR1:InvalidHex
E:0000:0000:0000:0000:InvalidHex