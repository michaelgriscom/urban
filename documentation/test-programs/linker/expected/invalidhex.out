H:InvalidHex:0000:5010:0100:2012145,15,04,36:URBAN-LLM:0019:InvalidHex
E:0002:0000:InvalidHex
